scripts/install-deps.sh

```diff
@@ -72,7 +72,7 @@ install_githooks=false
 
 du_test_data_dir_path="/tmp/adu/"
 # DO Deps
-default_do_ref=v1.0.0
+default_do_ref=develop
```

Comment:
for develop, it can be develop.  If destined for main, it should be either a commit hash or tag so `<commit hash>` is better than `main`, which is a potential supply chain issue since it would always be pulling down latest main, which always has potential to break things (or worse).